src/cow-react/modules/application/containers/App/styled.ts

```diff
@@ -18,7 +18,7 @@ export const HeaderWrapper = styled.div`
 
 export const FooterWrapper = styled(HeaderWrapper)`
   z-index: 1;
```

Comment:
I think you will have to set this z-index to `-1` if you want cows SVG behind the content.
But don't know if that has some hidden consequences. 
![Screenshot from 2022-10-21 17-31-23](https://user-images.githubusercontent.com/34926005/197234693-47b596e0-2b8b-4851-9164-45fe32685950.png)